docs/configurations.md

```diff
@@ -0,0 +1,133 @@
+# Configurations
```

Comment:
Is this created automatically or manually?

Reply:
Manually but I can work on some automation because I am already tired of copy pasting twice.